src/patternfly/components/Page/page.scss

```diff
@@ -247,7 +254,6 @@ $pf-page-v6--height-breakpoint-map: build-breakpoint-map("base", "sm", "md", "lg
   &.pf-m-limit-width {
     display: flex;
     flex-direction: column;
-    padding: 0;
 
     > .#{$page}__main-body {
       flex: 1;
```

Comment:
I wonder if we can remove this? It's causing this issue of the parent section fills the available space, the `main-body` elements will all grow to fill the space

<img width="2489" alt="Screenshot 2024-06-14 at 10 45 58 AM" src="https://github.com/patternfly/patternfly/assets/35148959/3203ab60-dfa1-4317-9cc7-fd19f0f41c7d">